### ISSUE TITLE
Subset linelist data with outcome by LinelistData.closed()

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#362]
## Summary of this new feature
Create new method of `LinelistData` class to subset the records by outcome.

## (Optional) Solution
Add `LinelistData.closed(country, province=None, outcome="Recovered")` method.
`outcome` will be selected from "Recovered" (default) and "Fatal".

**Comment by lisphilar** [comment]
Usage will be
```Python
linelist = data_loader.linelist()
df = linelist.closed("China", outcome="Recovered")
```
`outcome` can be selected from "Recovered" and "Fatal".

**Comment by lisphilar** [comment]
This was included in development version 2.12.0-beta.

**Comment by Inglezos** [comment]
Because of our discussion in issue #357, I removed with pull request #364 the arguments `country` and `province`, since the linelist data for each country are either missing or too little. I think it is better to use the complete global linelist data, in order to extract closing period for example.

**Comment by lisphilar** [comment]
WIth #364 by @Inglezos , usage of `LinelistData.closed()` was changed to `LinelistData.closed(outcome="Recovered")` in version 2.12.0-gamma. `country` and `province` arguments were removed.
